### PR TITLE
API Release Note for 2021.03.3

### PR DESCRIPTION
**Pull request by IanHakes-okta**
## Description:
- **What's changed?** 
API Release Notes 
- **Is this PR related to a Monolith release?** 
2021.03.3
### Resolves:

* [OKTA-380221](https://oktainc.atlassian.net/browse/OKTA-380221)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ title: Okta API Products Release Notes
...
 
+### Weekly Release 2021.03.3
+
+| Change                                                    | Expected in Preview Orgs |
+| --------------------------------------------------------- | ------------------------ |
+| [Bugs fixed in 2021.03.3](#bugs-fixed-in-2021-03-3)       | March 24, 2021           |
+
+#### Bugs fixed in 2021.03.3
+
+* When an OAuth2 request was made with an access token instead of a required ID token, the error message didn't clarify that the request was made with the wrong token.
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2021/index.md:15]: as per our style, no bullet for singular bug: https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/1311737785/Dev+Docs+-+Release+Notes+Process+API+Dev+Platform#Write-the-Blurbs

**Review by brianduffield-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 2]
```diff
@@ -4,6 +4,16 @@ title: Okta API Products Release Notes
 
 ## March
 
+### Weekly Release 2021.03.3
+
+| Change                                                    | Expected in Preview Orgs |
+| --------------------------------------------------------- | ------------------------ |
+| [Bugs fixed in 2021.03.3](#bugs-fixed-in-2021-03-3)       | March 24, 2021           |
```
[the remark this answers is not in the document]
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2021/index.md:11]: Fixed

[the remark this answers is not in the document]
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2021/index.md:11]: Fixed (and in the link too)

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2021/index.md:15]: I - ugh - still disagree with this style, but change made. Good eye!

**Pull request merged**